simplefs/simplefs.go

```diff
@@ -117,3 +232,238 @@ func (k *SimpleFS) SimpleFSGetOps(_ context.Context) ([]keybase1.OpDescription,
 func (k *SimpleFS) SimpleFSWait(_ context.Context, opid keybase1.OpID) error {
 	return errors.New("not implemented")
 }
+
+// remotePath decodes a remote path for us.
+func remotePath(path keybase1.Path) (ps []string, public bool, err error) {
+	pt, err := path.PathType()
+	if err != nil {
+		return nil, false, err
+	}
+	if pt != keybase1.PathType_KBFS {
+		return nil, false, errOnlyRemotePathSupported
+	}
+	raw := path.Kbfs()
+	if raw != `` && raw[0] == '/' {
```

Comment:
Pass for Windows `K:/private/foo` and Linux `/keybase/private/foo` just `/private/foo`. Just using slashed on all platforms makes sense.